### ISSUE TITLE
Get https://storage.googleapis.com/tilt-static-assets/v0.10.24/index.html: dial tcp [::1]:443: connect: connection refused

### ISSUE THREAD

**Issue by dealancer** [windmilleng/tilt#2688]
I am seeing following message in the browser when starting Tilt:

```
Get https://storage.googleapis.com/tilt-static-assets/v0.10.24/index.html: dial tcp [::1]:443: connect: connection refused
```

Any ideas why?

**Comment by maiamcc** [comment]
Sorry you're running into this issue! Can you navigate to that url in your browser? (The page will be blank, but will have a Tilt favicon.) Is it possible that it's being blocked by a firewall or something similar? (See also: #2608)

**Comment by dealancer** [comment]
@maiamcc,

https://storage.googleapis.com/tilt-static-assets/v0.10.24/index.html shows me Can't connect to the server error. I will later check it at home to figure out if it is a firewall issue or not.

**Comment by nicks** [comment]
I'm going to mark this as a duplicate of #2608 , about serving the web assets from offline. thanks for the report!
